Configuration/Policy.yaml

```diff
@@ -17,6 +19,10 @@ roles:
 
   'Neos.RedirectHandler.Ui:RedirectAdministrator':
     privileges:
+      - privilegeTarget: 'Neos.Neos:Backend.GeneralAccess'
+        permission: GRANT
+      - privilegeTarget: 'Neos.RedirectHandler.Ui:Backend.Module.Management'
```

Comment:
Why not simply use `Neos.Neos:Backend.Module.Management` here instead of a new custom privilege target?

Reply:
Tried that but that did not work 🤷‍♂️

Reply:
What happened when you did?
Did you also adjust the redirect after login setting for Neos?

Reply:
I recommend using `Neos.Neos:Backend.Module.Management` and `Neos.Neos:Backend.GeneralAccess` and that should work.